Sources/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -825,6 +827,24 @@ extension MiniAppScriptMessageHandler {
     }
 }
 
+//MARK: MiniApp GetHostAppThemeColors
+extension MiniAppScriptMessageHandler {
+    func getHostAppThemeColors(with callbackId: String) {
+        hostAppMessageDelegate?.getHostAppThemeColors(completionHandler: { (result) in
+            switch result {
+            case .success(let hostAppColors):
+                guard let hostAppColors = ResponseEncoder.encode(data: hostAppColors) else {
+                    self.executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: getMiniAppErrorMessage(MiniAppErrorType.hostAppError))
+                    return
+                }
+                self.executeJavaScriptCallback(responseStatus: .onSuccess, messageId: callbackId, response: hostAppColors)
+            case .failure(let error):
+                self.handleMASDKError(error: error, callbackId: callbackId)
```

Comment:
`handleMASDKError` shouldn't be used. Please use `prepareMAJavascriptError` instead

Reply:
l will check an update this.

Reply:
I have changed to use the recommended apis by updating error parameter to MiniAppJavaScriptError class object.